src/lang/lang.pri

```diff
@@ -1,28 +1,4 @@
 TS_FILES += $$files(qbittorrent_*.ts)
```

Comment:
This extra variable isn't really needed.

Reply:
qmake cannot handle correctly if I inline it.

Reply:
Only a configure warning:
```
RCC: Warning: No resources in 'qmake_qmake_qm_files.qrc'.
```
And lrelease is not invoked when building.

Reply:
>Are you sure you're doing this right?

Not the same way, but your way works! so the PR is updated.